prowler/providers/azure/services/monitor/monitor_storage_account_with_activity_logs_is_private/monitor_storage_account_with_activity_logs_is_private.py

```diff
@@ -0,0 +1,44 @@
+from prowler.lib.check.models import Check, Check_Report_Azure
+from prowler.providers.azure.services.monitor.monitor_client import monitor_client
+from prowler.providers.azure.services.storage.storage_client import storage_client
+
+
+class monitor_storage_account_with_activity_logs_is_private(Check):
+    def execute(self) -> Check_Report_Azure:
+        findings = []
+
+        for (
+            subscription_name,
+            diagnostic_settings,
+        ) in monitor_client.diagnostics_settings.items():
+            if not diagnostic_settings:
+                report = Check_Report_Azure(self.metadata())
+                report.subscription = subscription_name
+                report.resource_name = "Monitor"
+                report.resource_id = "Monitor"
+                report.status = "PASS"
+                report.status_extended = f"Blob public access disabled in storage account storing activity log in subscription {subscription_name} or not necessary."
+                findings.append(report)
+                continue
```

Comment:
Is this needed?

Reply:
When there are not diagnostic settings, the check must pass, this is for those cases, I did a test with no diagnostic settings and I noticed.

Reply:
I would say that this is not a PASS nor FAIL, there is no storage accounts where a finding can be applied.

Reply:
Okey